modules/inspecting-pod-and-container-logs.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * support/investigating-pod-issues.adoc
+
+[id="inspecting-pod-and-container-logs_{context}"]
+= Inspecting Pod and container logs
+
+Inspect Pod and container logs for warnings and error messages related to explicit Pod failures. Depending on policy and exit code, Pod and container logs remain available after Pods have been terminated.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* Your API service is still functional.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+
+.Procedure
+
+. Query logs for a specific Pod:
++
+----
+$ oc logs <pod_name>
+----
+
+
+. Query logs for a specific container within a Pod:
++
+----
+$ oc logs <pod_name> -c <container_name>
+----
+
+. Logs retrieved using the preceding `oc logs` commands are comprised  of messages sent to stdout within Pods or containers. Some applications might instead log messages to files contained in `/var/log` within a Pod or container.
```

Comment:
Since this is a step in a procedure, it should start with an actionable step if possible, instead of an explanation.

Reply:
Thank you, I will keep this in mind in future. I have now resolved this.